src/layout/download/furnace.js

```diff
@@ -51,6 +51,7 @@ const Furnace = ({ components, _ID, _body, _parseYaml, _relativeURL }) => {
 					<h3 className="furnace__title">
 						Select components
 						<button type="reset" className="js-furnace-clear furnace__clear au-btn au-btn--tertiary icon icon--right icon--close">Clear selections</button>
```

Comment:
```suggestion
						<button type="reset" className="js-furnace-clear furnace__clear au-btn au-btn--tertiary icon icon--right icon--close">Reset</button>
```

@sukhrajghuman Not sure on the wording for this one, Chameleon uses text: `Reset`